README.md

```diff
@@ -1,28 +1,27 @@
 ## Test Task for Elixir LiveView Developers
 
-The candidate should build a simple polling application using Phoenix LiveView. The application should allow users to create new polls, vote in polls, and see real-time updates of the poll results. The solution should not use any external dependencies, such as a database or disk storage, and should instead store all needed data in application memory. You are free to use any Elixir/Erlang library and any open-source CSS framework for the design.
+The candidate should build a simple polling application using Phoenix LiveView. The application should allow users to create new polls, vote in polls, and see real-time updates of the poll results. The solution must use an SQL database as storage to persist users, polls and poll results. You are free to use any Elixir/Erlang library and any open-source CSS framework for the design.
 
 ### Requirements
 
-1. All code should be shared via private Github repository. 
+1. All code should be shared via private Github repository.
 2. The solution should be built with Phoenix LiveView.
-3. Users should be able to create account by inserting their username.
-4. Users should be able to create new polls.
-5. Users should be able to vote in existing polls.
-6. Users should be able to see real-time updates of the poll results.
-7. User can only vote once in a single poll.
-8. Performance: users actions should not be blocking each other. User 1 actions should not be blocked by user 2 actions.
-9. You are free to use any Elixir/Erlang library and any open-source CSS framework for the UI.
-10. The application should not use any external dependencies, such as a database or disk storage. All needed data should be stored in application memory.
-11. The application should start with `mix phx.server` so it can be started locally.
-12. The application should be well-structured, and the code should be readable.
+3. The solution should use database as a persistent storage.
+4. Users should be able to create account by inserting their username.
+5. Users should be able to create new polls.
+6. Users should be able to vote in existing polls.
+7. Users should be able to see real-time updates of the poll results.
+8. User can only vote once in a single poll.
+10. You are free to use any Elixir/Erlang library and any open-source CSS framework for the UI.
+12. The application should start with `mix phx.server` so it can be started locally.
+13. The application should be well-structured, and the code should be readable.
 
 ### Deliverables
 
 Please provide a link to a GitHub repository containing your solution to the above task. Your solution should include:
 
 1. A Phoenix LiveView application that meets the above requirements.
-2. A README file that explains how to set up and run the application.
+2. A README file that explains how to set up and run the application. This includes any kind of set up instructions required for the database.
```

Comment:
Maybe it makes sense to specify that they should use docker for example. To avoid a case when they expect you to have it installed locally or some other weird setup.